backtrace-library/src/main/cpp/backtrace-native.cpp

```diff
@@ -294,6 +295,10 @@ namespace /* anonymous */
     bool InitializeLocalClientSideUnwinding(JNIEnv* env) {
         // Initialize a shared memory region.
         static const char *buffer = static_cast<char *>(buffer_create());
```

Comment:
Why is this buffer `const char *` ? Did you mean `static char *const buffer` ?

Reply:
do we have a specific naming style for globals? I think we should make it extra obvious that this is a `static` variable.

Reply:
Actually, why do we want a `static` buffer?